Orange/canvas/application/errorreporting.py

```diff
@@ -38,6 +38,20 @@ class OWWidget: pass
 log = logging.getLogger()
 
 
+def get_developer_id():
+    """Check if developer provided his name for Sentry reports."""
+    try:
+        from Orange.misc.environ import data_dir_base
+        fname = os.path.join(data_dir_base(), 'Orange', 'developer_id.txt')
+        if os.path.exists(fname):
+            with open(fname, 'r') as f:
+                username = f.readline().strip()
+                return username
+    except Exception:
```

Comment:
`except Exception` should always be avoided if possible.
Here you check if the file exists before doing anything, so users should be safe. And if developers manage to break something with a strange file, we probably want to see the error.